### PR TITLE
fixes overflow on mobile and re-arranged footer menu

### PR DESCRIPTION
**Pull request by Adebesin-Cell**
fixes overflow on mobile and re-arranged footer menu

Before:

https://user-images.githubusercontent.com/32770520/216031107-c0bca727-751f-46dd-af2f-838014fac87a.png

After:

localhost_4000_wiki_iqwiki (1)
localhost_4000_wiki_iqwiki

fixes https://github.com/EveripediaNetwork/issues/issues/1003

### CODE REVIEW

**Review by OleanjiKingCode — APPROVED** [review]

File: src/styles/global.css [hunk 1]
```diff
@@ -5,6 +5,10 @@ html {
   scroll-behavior: smooth;
 }
 
+body {
+  overflow-x: hidden;
+}
+
```
Comment (Royal-lobster) [inline src/styles/global.css:11]: @Adebesin-Cell you can add it to footer container if needed but for whole body it can be problematic.

Comment (Softdev1) [reply src/styles/global.css:11]: can we have it implemented @Adebesin-Cell

Comment (Adebesin-Cell) [reply src/styles/global.css:11]: Alright @Softdev1 @Royal-lobster 🙌

**Comment by Adebesin-Cell** [comment]
@Softdev1 Fixed 🙌

**Review by Softdev1 — APPROVED** [review]

**Pull request merged**